sample-java/src/main/java/social/bigbone/sample/GetHomeTimelineWithFiltering.java

```diff
@@ -61,8 +65,14 @@ public static void main(final String[] args) throws BigBoneRequestException {
                     System.out.println("* WARNING * status by " + name + " matches one or more of your filters");
                     System.out.println("********************************************************************************");
                 } else {
-                    final String content = status.getContent();
-                    System.out.println(name + " posted: " + content);
+                    if (shouldBlur.get()) {
+                        System.out.println("********************************************************************************");
+                        System.out.println("* WARNING * status by ~~~~~~~~~~~~~~~~~~~~~~ matches one or more of your filters");
```

Comment:
Shouldn’t `by` be followed by a user name or is there some dark magic involved here that replaces the tildes with a name? 😬

Reply:
That was my poor man's attempt at mimicking post blurring in text form. ;) If that is too obscure for the sample, I can replace it with something else.

Reply:
Haha 😅 I didn’t get that at first but its whimsical enough for me to keep it.